content/blog/usethis-2-0-0/index.Rmd

```diff
@@ -0,0 +1,258 @@
+Those who prefer SSH may want to set this option in `.Rprofile` going forward.
+
+## Pull request helpers
+
+The team that maintains the tidyverse and r-lib packages makes heavy use of GitHub pull requests for managing internal and external contributions.
+The `pr_*()` family of functions supports pull request workflows, for maintainers and contributors.
+This family has gained a couple of functions and some existing functions are more helpful.
+
+New pull request functions:
+
+* `pr_resume()` resumes work on an existing local PR branch. It can be called
+  argument-less, to select a branch interactively.
+  
+    ```{r, eval = FALSE}
+    > pr_resume()
+    ℹ No branch specified ... looking up local branches and associated PRs
+    Which branch do you want to checkout? (0 to exit) 
+    
+    1: avalcarcel9-add_use_author --> #833 ('@avalcarcel9'): Add use author
+    2:                 latex-hell
+    3:                     holder
+    4:            patch-for-withr
+    
+    Selection: 
+    ```
+  
+* `pr_forget()` abandons a PR you initiated locally or fetched from GitHub. It
+  only does local clean up and will never delete a remote branch or close a PR.
+  
+`pr_fetch()` and `pr_view()` also present an interactive choice, when the target PR is not specified or implied.
+
+## Other goodies
+
+Here's a selection of other new features:
+
+  * A default Git branch named `main` now works. `git_branch_default()` is a new
+    function that tries to discover the default branch from the local or remote
+    Git repo. Internally, it is used everywhere that we previously assumed a
+    default branch named `master`.
+  * `use_github_pages()` and `use_tidy_pkgdown()` are great for turning on
+    GitHub Pages and for using GitHub Actions to build and deploy a pkgdown
+    site.
+  * usethis knows that RStudio >= 1.3 stores user preferences in a file, which
+    means that `use_blank_slate()` can be used to opt in to the "never save to
+    or restore `.RData`" lifestyle, globally, for a user.
+  * `browse_package()` and `browse_project()` are new additions to the
+    `browse_*()` family that let the user choose from a list of URLs derived
+    from local Git remotes and DESCRIPTION (local or possibly on CRAN).
+    
+    ```{r, eval = FALSE}
+    > browse_package()
+    Which URL do you want to visit? (0 to exit) 
+    
+    1: https://github.com/r-lib/usethis        ('origin' remote)
+    2: https://github.com/avalcarcel9/usethis  ('avalcarcel9' remote)
+    3: https://usethis.r-lib.org               (URL field in DESCRIPTION)
+    4: https://github.com/r-lib/usethis        (URL field in DESCRIPTION)
+    5: https://github.com/r-lib/usethis/issues (BugReports field in DESCRIPTION)
+    
+    Selection: 
+    ```
+  * The `use_*_license()` functions have gotten a general overhaul and also now
+    work for projects, not just for packages.
```

Comment:
@Hadley sorry I got very caught up in describing all the Git/GitHub changes 😂. Would you prefer that this have a subsection?

Reply:
I don't think it needs its own section, but it would be useful to make a bit more prominent, and mention https://r-pkgs.org/license.html.